functions/pipeline.py

```diff
@@ -0,0 +1,467 @@
+'''
+Author: Alexandra Lee
+Date Created: 11 March 2020
+
+Scripts to run entrire pipeline from processing data, training VAE, running simulation experiments and
+generating output figures
+'''
+
+import os
+import sys
+import ast
+import pandas as pd
+import numpy as np
+import random
+import math
+import gc
+from sklearn import preprocessing
+
+from joblib import Parallel, delayed
+import multiprocessing
+
+import warnings
+warnings.filterwarnings(action='ignore')
+
+sys.path.append("../../")
```

Comment:
```suggestion
```

Reply:
This line is unnecessary since `pipeline.py` is already in the functions directory

Reply:
Good catch